doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>I like this, maybe it's a better name than _Theoretical overview_ and we should use it throughout the entire library.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>